spec/models/comment_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
```

Comment:
## Add unit specs for all of your models' methods and validations:
- You have done a good Job. Please try to restructure your code with something like this:
- 
![Capture](https://user-images.githubusercontent.com/63972670/192060327-1a201918-35fb-4819-bfe0-c9f70a2d3738.JPG)
- Write code that corresponds to what I pasted
- Kindly restructure like, post, comment, user if possible
-